pkg/asset/machines/master.go

```diff
@@ -134,6 +137,20 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 		}
 		openstack.ConfigMasters(machines, clusterID.InfraID)
 	case nonetypes.Name, vspheretypes.Name:
+		// machines are managed directly by users
+		return nil
```

Comment:
nit: this change shouldn't exist, the master looks like:
https://github.com/openshift/installer/blob/master/pkg/asset/machines/master.go#L136